install/kubernetes/helm/istio/charts/kiali/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 description: Kiali is an open source project for service mesh observability, refer to https://github.com/kiali/kiali for detail.
 name: kiali
-version: 0.5.1
-appVersion: 0.5.1
+version: 1.0.1
+appVersion: 1.0.1
```

Comment:
I think Kiali should be 0.6.0?

Reply:
Created a PR https://github.com/istio/istio/pull/8092 to update Kiali version to v0.6.0